### PR TITLE
Bump up the version for rocm-6.2.4 release

### PR DESCRIPTION
**Pull request by srekolam**

### CODE REVIEW

**Review by afzpatel — APPROVED** [review]

**Review by renjithravindrankannath — APPROVED** [review]
LGTM

**Comment by srekolam** [comment]
@tldahlgren  this PR is ready for review and merge.can you please let us know if you have any inputs.

**Comment by srekolam** [comment]
@alalazo  this PR is ready for review, and merge.

File: var/spack/repos/builtin/packages/omniperf/package.py [hunk 1]
```diff
@@ -19,8 +19,9 @@ class Omniperf(CMakePackage):
 
     license("MIT")
 
-    version("6.2.1", sha256="c14cb73b9fe17a3cca31489a73e8ea49cb278093f8d64c433b1bac387445074a")
-    version("6.2.0", sha256="b2ad49324a07aef977833d62741509a5d799b92758db56a16b4ab5636b6e231e")
+    version("6.2.4", sha256="2230260fce0838583899f4969b936ca047b30985a0fffad276ea353232538770")
+    version("6.2.1", sha256="56b795d471adad8ee9d7025544269e23929da31524d73db6f54396d3aca1445a")
+    version("6.2.0", sha256="febe9011e0628ad62367fdc6c81bdb0ad4ed45803f79c794757ecea8bcfab58c")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/omniperf/package.py:24]: Not a blocker but would be good to know why these changed.

**Review by tldahlgren — APPROVED** [review]
Confirmed the new version directive sha256s/commit(s). Have a question but am not going to let it be a blocker.

**Pull request merged**